### ISSUE TITLE
Zoom to bounds of selected animals

### ISSUE THREAD

**Issue by zkuralt** [romunov/wolfexplorer#10]
Set map bounds to currently selected animals and siblings.

**Comment by romunov** [comment]
Ah crap, I mean to close 11, not this one.

**Comment by zkuralt** [comment]
fitBounds(~min(lng), ~min(lat), ~max(lng), ~max(lat)) should do the trick

**Comment by romunov** [comment]
Yes, but should be limited to a click. It would suck if you were knocked all over the map as you are selecting animals.
